server/src/api/EventRoutes.ts

```diff
@@ -207,4 +207,41 @@ eventRoutes.get<void, MyEventResponse>(
   }
 );
 
+eventRoutes.delete("/delete/:eventId", async (req, res) => {
```

Comment:
isClubAdmin middleware, remove the redundant logic you have checking if the user is club admin

Reply:
include the FE with this PR. you don't have to split prs by frontend and backend